pkgslog.go

```diff
@@ -36,6 +35,18 @@ func NewPackageHandler(h slog.Handler, packages map[string]slog.Level) *PackageH
 
 // Enabled reports whether the handler handles records at the given level.
 func (h *PackageHandler) Enabled(ctx context.Context, level slog.Level) bool {
+	// Scan our package log levels and see if we have any
+	// packages configured at the requested log level.
+	//
+	// If we do, we will be handling that log level
+	for _, pkgLevel := range h.packages {
+		if pkgLevel <= level {
+			return true
+		}
+	}
```

Comment:
Hi @jippi, thanks for pointing that out.

For better performance, I'd precompute the minimum log level in the constructor:
```go
        var minPkgLevel *slog.Level
	for _, pkgLevel := range packages {
		if minPkgLevel == nil || pkgLevel < *minPkgLevel {
			minPkgLevel = &pkgLevel
		}
	}
	return &PackageHandler{
		handler:     h,
		packages:    packages,
		minPkgLevel: minPkgLevel,
	}
```

And then compare with it in the Enabled method:
```go
        if h.minPkgLevel != nil && *h.minPkgLevel <= level {
		return true
	}
```

I used a pointer here to allow empty map arguments.